lectures/scalar_dynam.md

```diff
@@ -220,11 +220,10 @@ For example, in a later lecture {doc}`solow`, we will study the Solow-Swan growt
 ```{math}
 :label: solow_lom2
 
-k_{t+1} = s z k_t^{\alpha} + (1 - \delta) k_t
+k_{t+1} = s A k_t^{\alpha} + (1 - \delta) k_t
 ```
 
-Here $k$ is the per capita capital stock and $s, z, \alpha, \delta$ are positive
-parameters with $0 < \alpha, \delta < 1$.
+Here $k=K/L$ is the per capita capital stock, $s$ is the saving rate, $A$ is the total factor productivity, $\alpha$ is the capital share, and $\delta$ is the depreciation rate. All these parameter are positive and $0 < \alpha, \delta < 1$.
```

Comment:
This is nice, thanks @longye-tian .

Perhaps "All these..." should be on a new line to match the style manual.

Reply:
Got it!

I will revise it now ❤️